v2.js

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright (c) 2016, Yahoo Inc.
+Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
+*/
+
+var LIBS = {
+        colors:     require('css-color-names'),
```

Comment:
hehe... yeah I thought that that might be controversial. (I'm very fond of the LIBS container but I will change.)